apps/web/playwright/auth/delete-account.test.ts

```diff
@@ -11,6 +11,7 @@ test("Can delete user account", async ({ page, users }) => {
   await page.waitForSelector("[data-testid=dashboard-shell]");
 
   await page.goto(`/settings/profile`);
+  await page.fill("[data-testid=password]", "delete-me");
```

Comment:
- This is trying to fill the password before the modal opens
- The password is incorrect, it is generated per-test